src/posts/tracker/basics.ts

```diff
@@ -105,9 +105,9 @@ This table provides a fuller explanation for each of the Built-in Person Risk pr
    </td>
   </tr>
   <tr>
-   <td>Has 1/4/10 close contactwhose risk profile we don't know. No other risk activities.
+   <td>Has 1/4/10 close contacts whose risk profile we don't know. No other risk activities.
    </td>
-   <td>A known person who has close contact (including roommates) with 1/4/10 people who do not work with the public, and are in contact with the outside world (may be unmasked, indoors, or within 20 feet. Assumes all people are going on one grocery/errand run in the past week. 
+   <td>A known person who has close contact (including roommates) with 1/4/10 people who do not work with the public, and are in contact with the outside world (may be unmasked, indoors, or within 20 feet of others). Assumes all people are going on one grocery/errand run in the past week.
```

Comment:
Can we remove the "who do not work with the public" part?

Reply:
done